dictionary.py

```diff
@@ -237,14 +241,14 @@ def write_index(entries, dictionary_name, title, stream, respect_re_restr=True,
             entry.sentences.sort(reverse=True, key = lambda sentence: sentence.good_sentence)
             for sentence in entry.sentences:
                 stream.write(' <div class="sen">\n')
-                stream.write('  <span>' + sentence.japanese + '</span>\n')
+                stream.write(f"  <span>{sentence.japanese}</span>\n")
```

Comment:
f-strings is great, but for future reference, I'd prefer if we standardized on single quotes `'` strings, as this makes it easier to emit `"` in the generated XML/HTML